pkg/template/api/v1/types.go

```diff
@@ -14,15 +14,19 @@ type Template struct {
 	// Standard object's metadata.
 	kapi.ObjectMeta `json:"metadata,omitempty"`
 
-	// Objects is an array of objects to include in this template. Required.
+	// PostProcessMessage is an optional instructional message for the user
+	// processing this template (parameter substitution will be performed).
+	PostProcessMessage string `json:"postProcessMessage,omitempty"`
```

Comment:
PostProcess is too technical, I'd want someone who is a brand new user to figure it out.  Message is probably sufficient.